packages/fluentui/accessibility/src/behaviors/Toolbar/toolbarMenuItemBehavior.ts

```diff
@@ -27,8 +27,8 @@ export const toolbarMenuItemBehavior: Accessibility<ToolbarMenuItemBehaviorProps
 };
 
 export type ToolbarMenuItemBehaviorProps = {
-  /** Indicated if menu item has submenu. TODO: fix types for all behaviours */
-  menu?: any;
+  /** Indicated if menu item has submenu */
+  hasMenu?: boolean;
```

Comment:
would it make more sense just to intersect `Pick<MenuItemBehaviourProps, 'hasMenu'> ?

Reply:
This is a great suggestion, after checking MenuItemBehavior, I changed toolbarMenuItemBehaviorProp to:
```tsx
export type ToolbarMenuItemBehaviorProps = Omit<MenuItemBehaviorProps, 'vertical'>;
```